### PR TITLE
Made selecting a bone not clear the undo history

### PR DESCRIPTION
**Pull request by sts1skj**
I made it so that the undo history is not cleared when a bone is
selected.  I also tried to make sure that bone GUI elements update
at the right times.

### CODE REVIEW

File: src/program/OutfitStudio.cpp [hunk 1]
```diff
@@ -2003,6 +2002,15 @@ void OutfitStudioFrame::GetNormalizeBones(std::vector<std::string> *normBones, s
 	}
 }
 
+std::vector<std::string> OutfitStudioFrame::GetSelectedBones() {
+	std::vector<std::string> boneList;
+	wxArrayTreeItemIds selItems;
+	outfitBones->GetSelections(selItems);
+	for (int i = 0; i < selItems.size(); i++)
+		boneList.push_back(outfitBones->GetItemText(selItems[i]));
```
Comment (ousnius) [inline src/program/OutfitStudio.cpp]: Doesn't compile for me on my wxWidgets and MSVC++ (pushing wxString into std::string vector).

**Comment by sts1skj** [comment]
Hopefully commit 54b4005 fixes that compilation problem for MSVC++.

**Comment by ousnius** [comment]
@sts1skj That wasn't the right line. You can revert that, line 2010 is the one that doesn't compile.

**Comment by sts1skj** [comment]
Oops.  Hopefully I got it right this time.

**Pull request merged**